terms/WasAttributedTo

```diff
@@ -0,0 +1,24 @@
+{
+    "@context": "../contexts/base",
+    "@graph": [
+        {
+            "@id": "nidm:subject_id",
+            "schema:domainIncludes": {
+                "@id": "reproschema:WasAttributedTo"
+            }
+        },
+        {
+            "@id": "reproschema:Response",
+            "@type": [
+                "rdfs:Class",
+                "prov:Entity",
+                "nidm:DataElement"
```

Comment:
```suggestion
                "prov:Person"
```

Reply:
are you suggesting  this change for `@type` for `reproschema:Response`? or only in this specific file -`WasAttributedTo`?